pages.pt_BR/common/helm.md

```diff
@@ -0,0 +1,36 @@
+# helm
+
+> Helm é um gerenciador de pacores para Kubernetes.
+> Mais informações: <https://helm.sh/>.
+
+- Cria um chart do helm:
+
+`helm create {{chart_name}}`
+
+- Adiciona um novo repositório helm:
+
+`helm repo add {{repo_name}}`
+
+- Lista os repositórios helm:
+
+`helm repo list`
+
+- Atualiza os repositórios helm:
+
+`helm repo update`
+
+- Remova um repositório helm:
+
+`helm repo remove {{repo_name}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers in all page.

Reply:
I changed. Please take a look when possible.

Reply:
@caduvieira Plase apply the same change to the pages `jar` and `kill`

Reply:
Sorry my bad. Now it's done for all